client/app/scripts/liveblog-settings/module.js

```diff
@@ -0,0 +1,77 @@
+(function() {
+    'use strict';
+    LiveblogSettingsController.$inject = ['$scope', 'api', '$location', 'notify', 'gettext', '$q'];
+    function LiveblogSettingsController($scope, api, $location, notify, gettext, $q) {
+        //prep the settings
+        $scope.liveblogSettings = {'language': {}, 'theme': {}};
+        $scope.settingsLoading = true;
+        api.languages.query().then(function(data) {
+            $scope.languages = [];
```

Comment:
I don't think you need to instantiate a new array here, You can assign `data._items` directly to `$scope.languages`.